bin.src/ci_imsim_run.py

```diff
@@ -167,9 +134,9 @@ def run(self, currentState: BuildState):
             "run",
             "-j", str(self.arguments.num_cores),
             "-b", self.runner.RunDir,
-            "-i", HIPS_COLLECTION,
+            "-i", COLLECTION,
```

Comment:
You are changing this here, but not removing HIPS_COLLECTION above. Are you sure all tests will do the right thing or things will end up in the right place?